Add migration guide for 1.8 (#6515)
kyma-project/kyma: docs/migration-guides/1.7-1.8.md

# Migrate from v1.7 to v1.8

>**TIP:** To learn more about upgrading Kyma, read [this](https://kyma-project.io/docs/master/root/kyma/#installation-upgrade-kyma) document.

## Monitoring

> We have increased the size of Persistent Volume Claims (PVC) for Prometheus to 10GB to prevent it from running out of disk space. Unfortunately, the standard upgrade procedure does not resize the volumes automatically, as the volume expansion through StatefulSets is not supported.

Reviewer: ```suggestion
We have increased the size of Persistent Volume Claims (PVC) for Prometheus to 10GB to prevent it from running out of disk space. The automatic upgrade procedure does not resize the volumes automatically, as the volume expansion through StatefulSets is not supported. As a result, you must trigger the Prometheus PVC size change manually.
```

> To make sure PVC size is 10GB, perform the following steps:

Reviewer: ```suggestion
Follow these steps:
```

> 1. Increase PVC size:

Reviewer: ```suggestion
1. Increase the PVC size:
```

```

> kubectl patch pvc prometheus-monitoring-db-prometheus-monitoring-0 -n kyma-system -p '{"spec":{"resources":{"requests":{"storage":"10Gi"}}}}'

Reviewer: Are you sure the Pod name is always the same?

```
2. Delete the `prometheus-monitoring-0` Pod:  

```
kubectl delete pod prometheus-monitoring-0 -n kyma-system
```

> The Pod will recreate with the PVC size already increased.

Reviewer: ```suggestion
The Pod will recreate with the PVC size increased.
```
